ttnn/cpp/ttnn/tensor/layout/tensor_layout.cpp

```diff
@@ -36,6 +36,12 @@ Alignment legacyShapeToAlignment(const ttnn::Shape& shape) {
         values[i] = legacy_padded_shape[i] * values[i + 1];
     }
 
+    for (auto& value : values) {
```

Comment:
will this conflict with @TT-BrianLiu 's , change?

Reply:
I'm unsure. I would say this PR in general is likely to conflict with Brian's PR. Let's merge his PR first and I'll resolve the conflicts

Reply:
I merged. Try rebasing? I don't think it's an issue